docs/code-snippets/outlook-snippets.yaml

```
@@ -513,6 +513,28 @@ Office.EmailAddressDetails#recipientType:member:
     requiredAttendees.forEach(function (requiredAttendee) {
         console.log("Attendee " + requiredAttendee.displayName + ": " + requiredAttendee.recipientType);
     });
+Office.EmailUser:interface:
+  - |-
+    // Add recipients to the To field of an email.
+    const recipients = [
```

Since it's not immediately obvious where the EmailUser object is, consider using an explicit type. (Please double check the syntax for the array assignment in case I'm missing something.)
```suggestion
    const recipients:  Office.EmailUser[] = [
```